ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -204,6 +238,36 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-hypershift
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: aws-ipi-ovn-ipsec-f28-obo
```

Comment:
The `obo` keyword are added for `Cluster_Observability` team (cc @liangxia to confirm), and the install workflow is using `cucushift-installer-rehearse-aws-ipi-ovn-ipsec`, if you want to cover it in upgrade ci, suggest to remove `obo` keyword.

Reply:
both removed